src/08_comprehensions.py

```diff
@@ -2,40 +2,45 @@
 List comprehensions are one cool and unique feature of Python.
 They essentially act as a terse and concise way of initializing
 and populating a list given some expression that specifies how
-the list should be populated. 
+the list should be populated.
 
 Take a look at https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions
 for more info regarding list comprehensions.
 """
 
 # Write a list comprehension to produce the array [1, 2, 3, 4, 5]
 
-y = []
+y = [num for num in range(6) if num > 0]
```

Comment:
This works, but range also can take a start parameter, like... `range(1,6)` so it starts at 1.